control/veda_file_discovery.py

```diff
@@ -125,162 +127,199 @@ def about_video_validate(self, meta, key):
 
         reset_queries()
 
-    def studio_s3_ingest(self):
+    def move(self, key, destination_dir):
         """
-        Ingest files from studio upload endpoint
+        Moves an S3 key file to destination directory.
```

Comment:
`Move an S3 file to another directory on same S3 bucket` or better docstring which tells that this is an S3 to S3 move operation.